mlflow/langchain/__init__.py

```diff
@@ -761,21 +759,27 @@ def _load_pyfunc(path):
 
 def _load_model_from_local_fs(local_model_path):
     flavor_conf = _get_flavor_configuration(model_path=local_model_path, flavor_name=FLAVOR_NAME)
-    _add_code_from_conf_to_system_path(local_model_path, flavor_conf)
     if _CODE_CONFIG in flavor_conf:
         path = flavor_conf.get(_CODE_CONFIG)
         flavor_code_config = flavor_conf.get(FLAVOR_CONFIG_CODE)
         if path is not None:
-            code_path = os.path.join(
+            config_path = os.path.join(
                 local_model_path,
                 flavor_code_config,
                 os.path.basename(os.path.abspath(path)),
             )
         else:
-            code_path = None
+            config_path = None
+
+        code_path = os.path.join(
+            local_model_path,
+            flavor_code_config,
+            os.path.basename(os.path.abspath(flavor_conf.get(_CODE_PATH))),
```

Comment:
```suggestion
            os.path.basename(flavor_conf.get(_CODE_PATH)),
```

Is `os.path.abspath` necessary?